ui/components/editor-channel-dialog.vue

```diff
@@ -359,6 +364,55 @@ export default {
       this.resetChannelForm();
     },
 
+    onChannelNameChanged(name) {
+      if (!this.areCapabilitiesChanged) {
+        const cap = this.channel.capabilities[0];
+        const changeCapabilityType = this.$refs.capabilities[0].$refs.capabilityTypeData.changeCapabilityType;
+        const singleColors = this.properties.capabilityTypes.ColorIntensity.properties.color.enum;
+
+        const matchingColor = singleColors.find(
+          color => name.toLowerCase().includes(color.toLowerCase())
+        );
+        if (matchingColor) {
+          cap.type = `ColorIntensity`;
+          changeCapabilityType();
+          cap.typeData.color = matchingColor;
```

Comment:
Since `changeCapabilityType` happens asynchronously, swap the two lines to make people reading the code have the order of instructions correct in their minds instantly.